portfolio/src/main/webapp/community.css

```diff
@@ -0,0 +1,11 @@
+
+
+body, html {
+    background-image: url(/images/community-image.JPG);
+    background-repeat: no-repeat;
+    background-size: cover;
```

Comment:
I think you can shorthand these.

Reply:
Whenever I shorthand it breaks my image, but I'll leave this unresolved so I can experiment a bit later

Reply:
Figured out shorthand -- was missing color